libs/market-depth/src/lib/orderbook.spec.tsx

```diff
@@ -183,4 +183,53 @@ describe('Orderbook', () => {
     );
     expect(result.getByTestId('scroll').scrollTop).toBe(6 * rowHeight);
   });
+
+  it('should format correctly the numbers on resolution change', async () => {
+    const onClickSpy = jest.fn();
+    window.innerHeight = 11 * rowHeight;
+    const result = render(
```

Comment:
suggestion (non blocking): prefer not using the return value of `render` and use the direct import `screen` instead

Reply:
updated